buildlib/tools/common.sh

```diff
@@ -54,3 +58,16 @@ prepare_build() {
 	cd ${ucx_build_dir}
 	export PROGRESS=0
 }
+
+
+install_pgi() {
+	mkdir -p $HOME/bin
+	# HACK: bc tool need for build hpc_sdk , we will copy it from NFS share
+	cp $PGI_ROOT_DIR/../bc $HOME/bin
+	NVHPC_ACCEPT_EULA=accept NVHPC_INSTALL_DIR=$PGI_INSTALL_DIR NVHPC_SILENT=true $PGI_ROOT_DIR/install
+	export MODULEPATH=$MODULEPATH:$PGI_INSTALL_DIR/modulefiles
+}
+
+uninstall_pgi() {
+	rm -rf $PGI_INSTALL_DIR
+}
```

Comment:
why needed? docker..

Reply:
this dir have 6G size, and i think better remove after use
for avoid no space

Reply:
isn't it gone after docker exits? or it's still cached somewhere?